pkgs/build-support/php/build-composer-project.nix

```diff
@@ -89,6 +90,8 @@ let
           composerStrictValidation = previousAttrs.composerStrictValidation or true;
         });
 
+      passthru.updateScript = previousAttrs.passthru.updateScript or (if finalAttrs.composerRepository.composerLock == null then nix-update-script { } else null);
```

Comment:
This will clobber `passthru` attribute – remember `{passthru.updateScript = …;}` is syntactic sugar for { passthru = { updateScript = …; }; }`.